x-pack/filebeat/module/cef/log/config/input.yml

```diff
@@ -24,3 +24,22 @@ processors:
   - decode_cef:
       field: event.original
   - community_id:
+  - script:
+      lang: javascript
+      source: >-
+        function forcepoint_mappings(evt) {
```

Comment:
I liked what you had before, but just wanted a `when` condition added. The primary benefit to `when` is that it avoids invoking the javascript VM at all which I _assume_ is a performance improvement over doing the check in JS for non-forcepoint logs.